scripts/utils/user_interface_helpers.js

```diff
@@ -1,5 +1,13 @@
 module.exports = function (web3 = web3, artifacts = artifacts) {
-  const { promptUser } = require("./sign_and_send")(web3, artifacts)
+  const readline = require("readline")
+
+  const promptUser = function (message) {
+    const rl = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout,
+    })
+    return new Promise((resolve) => rl.question(message, (answer) => resolve(answer)))
```

Comment:
My best guess is that once the promise resolves that maybe [`rl.close()`](https://nodejs.org/api/readline.html#readline_rl_close) might need to be called?

Reply:
Or, the same `rl` can be reused for all `promptUser` calls?

Reply:
Would your second suggestion amount to moving the rl outside of the method. I think I will go  with your first suggestion though.

Reply:
They are just some ideas for getting it to work, I don't know enough about `readline` to know what the "correct" way to do it is.

Reply:
Actually, it turns out I only had to move the 

```
  const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
  })
```

outside of the function declaration!